neo/Prompt/Commands/SC.py

```diff
@@ -283,7 +284,25 @@ def execute(self, arguments):
                 print("Incorrect password")
                 return False
 
-            return InvokeContract(wallet, tx, Fixed8.Zero(), from_addr=from_addr)
+            n_fee = Fixed8.Zero()

```

Comment:
General remark for the code below. The information is printed _after_ we informed the user what their fees would be (from the testinvoke). This means when they enter their password they can get unexpected costs which we should avoid.